### ISSUE TITLE
GO:0099008 should not be linked to Swiss-Prot KW KW-1173

### ISSUE THREAD

**Issue by SIBvirus** [geneontology/go-ontology#15460]
Hello,
Could you please correct the following:
GO:0099008 (viral entry via permeabilization of inner membrane) which describes the entry of a non-enveloped virus into the cytoplasm of a host prokaryotic cell should not be cross-referenced to Swiss-Prot KW KW-1173 (Viral penetration via permeabilization of host membrane) because this KW is used for both the prokaryotic and the eukaryotic viruses. 
Thanks,
Chantal Hulo

**Comment by ukemi** [comment]
@ggeorghiou. I'm not sure who maintains the keyword mappings these days. Can you look into this or let me know who to contact? Thanks!

**Comment by ggeorghiou** [comment]
Hi @ukemi, that would be me. Looking into this now

**Comment by ggeorghiou** [comment]
Right, looking at the mappings for this keyword, it's mapped to two GO terms:

1) GO:0039665 - permeabilization of host organelle membrane involved in viral entry into host cell - https://www.ebi.ac.uk/QuickGO/term/GO:0046718
2) GO:0099008 - viral entry via permeabilization of inner membrane - https://www.ebi.ac.uk/QuickGO/term/GO:0099008

@SIBvirus - GO:0099008 covers the prokaryotes, while GO:0039655 covers everything but S. pombe and other fungi. I would perhaps suggest removing both terms and mapping the keyword to GO:0046718 - viral entry into host cell (https://www.ebi.ac.uk/QuickGO/term/GO:0046718) - however I feel this might be too generic since the keyword definition states viral penetration into the host cell - http://www.uniprot.org/keywords/KW-1173.

If you're happy to re-map this keyword to GO:0046718, I'll go ahead and do so. @sylvainpoux do you agree or have another suggestion?

**Comment by SIBvirus** [comment]
1. Mapping KW-1173 (Viral penetration via permeabilization of host membrane) to GO:0046718 - viral entry into host cell is right but "viral entry into host cell" is a very broad term... 

2. An exact matching to our KW-1173  would require the creation a GO term "viral entry via permeabilization of host membrane"  (which would include organellar + inner membrane in the definition). GO:0039665 and  GO:0099008 could thus be children of this term which itself would be a child of viral entry into host cell.

Just a question, I think you meant GO:0039665 in your comment? If the penetration is caused by a protein of the viral capsid, why do you need to exclude S. pombe and other fungi?

**Comment by ggeorghiou** [comment]
Actually I made a mistake, both GO terms that map to the keyword have the same taxon restraint to not allow annotation to pombe or other fungi. Why this is is a very good question and one that a pombe expert might be able to answer. @ValWood would you mind weighing in on why taxon restraints exist for GO:0039665 and GO:0099008? 

Furthermore, @ukemi what do you think of requesting a new term as @SIBvirus has suggested in her second point?

**Comment by ValWood** [comment]
These are viral terms? So we wouldn't use them to annotate fission yeast of budding yeast?
Have I understood the question?

**Comment by ggeorghiou** [comment]
Yes, they are viral terms. There are taxon restraints for these terms that say never in fungi and pombe, but not any other species.

This may be my ignorance on pombe and apologies for that, but do pombe never get viral infections? And if they do, are there any native proteins in pombe that a virus uses to help enter the cell? Or am I misunderstanding how the taxon restraints are applied in GO terms due to cloudy brain syndrome?

**Comment by ValWood** [comment]
These have probably  been added to prevent mappings occurring in the past.  
Virus's probably can infect   but the biological role of the pombe proteins isn't to help virus's enter the cell so we would not make these annotations ever......
We wouldn't annotate any viral terms to fission yeast.

v

**Comment by ggeorghiou** [comment]
Cheers Val for clearing that up!

**Comment by ukemi** [comment]
@SIBvirus  can we get a reference, definition and parent for the new term request. I have modified this ticket accordingly.

**Comment by SIBvirus** [comment]
Sorry for the late answer. I tried first to find a reference that would describe both the permeabilization of the endosomal membrane and from the inner membrane in bacteria. Unfortunately, research on eukaryotic and prokaryotic viruses are most of the time two separate fields.

I can propose you the following references: 

Permeabilization of the endosome (and possibly from the Golgi or ER for some viruses):
PubMedID: 25055856

Permeabilization of the inner membrane (although not very clear... the authors talk about an endocytosis-like mechanism, although it cannot be endocytosis as the host is a bacteria):
PubMedID: 20427561


By reading on this topic again, I see some inconsistencies in the GO hierarchy:

"GO:0099006 viral entry via permeabilization of endosomal membrane" is not linked to  GO:0039665 permeabilization of host organelle membrane involved in viral entry into host cell"

What is the difference between "GO:0075502 endosome membrane permeabilization involved in viral entry into host cell"  and "GO:0099006 viral entry via permeabilization of endosomal membrane"? I think these two are synonyms

In order to display a clear view of this process, I would propose the following hierarchy:

1. Viral entry into host cell (GO:0046718)
	      
2. Viral penetration via permeabilization of host membrane (new term)

3. Viral entry via permeabilization of inner membrane (GO:0099008)

3. Permeabilization of host organelle membrane involved in viral entry into host cell (GO:0039665)

4. Viral entry via permeabilization of endosomal membrane (GO:0099006) (synonym=endosome membrane permeabilization involved in viral entry into host cell GO:0075502)

                Viral entry into host cell (GO:0046718)
				 ¦
        Viral penetration via permeabilization of host membrane	-> linked to KW-1173      
		   ¦		                       ¦
	      GO:0039665           GO:0099008
		   ¦
	      GO:0099006
	 (synonym GO:0075502)


Definition of the new term:

Host membrane permeabilization triggered by an interaction between the bilayer of a host membrane and a membrane-penetration capsid protein. Results in release of the virus contents into the host cell cytoplasm. 

I hope this helps
Chantal

**Comment by pgaudet** [comment]
Created new term: 
+[Term]
+id: GO:0140267
+name: viral entry via permeabilization of host membrane
+namespace: biological_process
+def: "The entry of a virus into the cytoplasm of a host cell, triggered by an interaction between the bilayer of a host membrane and a membrane-penetration capsid protein. Results in release of the virus contents into the host cell cytoplasm." [PMID:20427561, PMID:25055856]
+is_a: GO:0046718 ! viral entry into host cell
+created_by: pg
+creation_date: 2018-09-25T14:33:03Z

DO NOT ANNOTATE (it's a grouping term)
----
Added children: 
* GO:0099008 'viral entry via permeabilization of inner membrane'
* GO:0099006 'viral entry via permeabilization of endosomal membrane'


-- 
also added 'part of' some 'viral entry via permeabilization of host membrane' to 
* 'endosome membrane permeabilization involved in viral entry into host cell'
* 'macropinosomal membrane permeabilization involved in viral entry into host cell'
* 'permeabilization of host organelle membrane involved in viral entry into host cell'

@SIBvirus let me know if this is not what you wanted. 

Thanks, Pascale
